### PR TITLE
Implement new flipper control

### PR DESCRIPTION
**Pull request by Tom-Willemsen**
https://github.com/ISISComputingGroup/IBEX/issues/4871

### CODE REVIEW

File: MezfliprSup/Mezflipr_single.db [hunk 1]
```diff
@@ -14,141 +15,160 @@ record(ai, "$(P)$(NAME):COMPENSATION") {
...
+    field(FLNK, "$(P)$(NAME):CURRENT_ANALYTICAL:SP:_SEND")
 }
 
-record(stringout, "$(P)$(NAME):FILENAME:SP") {
-    field(DESC, "$(NAME) Filename SP")
-    field(OUT, "@Mezflipr.proto setFile($(CODE)) $(PORT)")
+record(bo, "$(P)$(NAME):CURRENT_ANALYTICAL:SP:_SEND") {
+    field(OUT, "@Mezflipr.proto setCurrentAnalytical($(P)$(NAME)) $(PORT)")
     field(DTYP, "stream")
+}
```
Comment (DominicOram) [inline MezfliprSup/Mezflipr_single.db:134]: As discussed, try doing this as one record or put a note on stream device gotchas on wiki.

**Review by DominicOram — CHANGES_REQUESTED** [review]
Looks good, minor comment

File: MezfliprSup/Mezflipr_single.db (hunk 1 above)
Comment (Tom-Willemsen) [reply MezfliprSup/Mezflipr_single.db:134]: @FreddieAkeroyd and I tried this again with both `lso` and writing waveforms directly to streamdevice but could not get either one to work. I've documented how to do this at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Stream-Device-Tips-and-Tricks

Comment (DominicOram) [reply MezfliprSup/Mezflipr_single.db:134]: Could you also post on tech-talk as it looks like this should work from documentation?

**Review by DominicOram — APPROVED** [review]

**Pull request merged**